sim/dex-data.ts

```diff
@@ -52,7 +52,7 @@ export class Tools {
 	}
 	/* eslint-enable @typescript-eslint/prefer-optional-chain */
 }
-const toID = Tools.getId;
+const toID = Tools.toID;
```

Comment:
We might want to consider not having a `Tools` class at all. `toID` and `getString` can probably just be top-level.

Reply:
Heck, there's a good argument for moving `getString` to `lib/`.

Reply:
Honestly, can't they both be in lib/utils? both of them are pretty basic non-pokemon related functions.

Reply:
`toID` is PS-specific. `lib/utils` is intended to be more general-purpose.